### PR TITLE
review backup docs

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- replace velero instructions with link to velero docu
- merge remaining backup docs into 1 file
- update Gardener instructions: 
  - Azure supports CSI as of Kubernetes 1.21
  - add 4th step
- remove non-cloud-provider instructions because instructions for cloud providers are now the default/only way to go

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/11129

### CODE REVIEW

**Review by suleymanakbas91 — APPROVED** [review]

File: docs/04-operation-guides/operations/10-backup-kyma.md [hunk 1]
```diff
@@ -1,29 +1,232 @@
...
+  AKS
+  </summary>
+
+### Prerequisites
+
+  The minimum supported Kubernetes version is 1.17.
+
+### Steps
+
+  1. [Install the CSI driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-csi-driver-master.md).
```
Comment (mmitoraj) [inline docs/04-operation-guides/operations/10-backup-kyma.md:50]: It is a broken link.

File: docs/04-operation-guides/operations/10-backup-kyma.md [hunk 2]
```diff
@@ -1,29 +1,232 @@
...
+    dataSource:
+      name: snapshot
+      kind: VolumeSnapshot
+      apiGroup: snapshot.storage.k8s.io
+  ```
+
+  </details>
+ </div>
+
+## Create periodic snapshot job
```
Comment (mmitoraj) [inline docs/04-operation-guides/operations/10-backup-kyma.md:140]:
```suggestion
## Create a periodic snapshot job
```
Jobs always seemed countable to me, what do you think?

File: docs/04-operation-guides/operations/10-backup-kyma.md (hunk 1 above)
Comment (NHingerl) [reply docs/04-operation-guides/operations/10-backup-kyma.md:50]: Weird, it works from github and when I copy + paste the URL. It's just broken in the netlify preview.  How???

File: docs/04-operation-guides/operations/10-backup-kyma.md (hunk 2 above)
Comment (NHingerl) [reply docs/04-operation-guides/operations/10-backup-kyma.md:140]: Fully agree, thanks for spotting this!

File: docs/04-operation-guides/operations/10-backup-kyma.md (hunk 1 above)
Comment (NHingerl) [reply docs/04-operation-guides/operations/10-backup-kyma.md:50]: Investigation with https://sap-btp.slack.com/archives/C01RAHB7D25/p1627558780003700?thread_ts=1627468836.000800&cid=C01RAHB7D25

**Pull request merged**